api-authorization/server/lib/authorization-middleware.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-unused-vars -- Remove me */
+import jwt from 'jsonwebtoken';
+import { ClientError } from './client-error.js';
+
+export function authMiddleware(req, res, next) {
+  /* your code here */
+  const authHeader = req.get('Authorization');
+  const token = authHeader.split('Bearer ')[1];
```

Comment:
This has a bug. What happens if there is no authorization header?

Reply:
Fixed